### DISCUSSION TITLE
Install CLI

### DISCUSSION THREAD

**Discussion by Angelk90** [zed-industries/zed#6678] [Feature Requests]
Hi, what exactly is install cli for?

**Comment by jansol** [comment]
It's so that you can do `zed myfile.txt` in Terminal.app (or whatever terminal you use) to make Zed open `myfile.txt` (or whatever arbitrary file you want to edit). Otherwise you would have to something like `open -a Zed.app --args myfile.txt`.

**Reply by Angelk90** [reply]
@jansol : I understand, some considerations:

1) What if I wanted to remove it?
2) If the cli is installed, if I click it again shouldn't it tell me that it is already installed?

<img width="416" alt="Screenshot 2023-12-13 alle 10 41 19" src="https://github.com/zed-industries/community/assets/20476002/310a5c67-40da-436a-a330-e88005dbed04">

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
